README.md

```diff
@@ -982,14 +983,18 @@ Please refer to the linked specific documents when looking for information regar
 
 [Top](#top)
 
-##<a name="section9"></a>Contact
+##<a name="section9"></a>Reporting issues and contact information
+There are several channels suited for reporting issues and asking for doubts in general. Each one depends on the nature of the question:
 
-Francisco Romero Bueno (francisco.romerobueno@telefonica.com) **[Main contributor]**
-<br>
-Fermín Galán Márquez (fermin.galanmarquez@telefonica.com) **[Contributor and Orion Context Broker owner]**
-<br>
-Germán Toro del Valle (german.torodelvalle@telefonica.com) **[Contributor]**
-<br>
-Iván Arias León (ivan.ariasleon@telefonica.com) **[Quality Assurance]**
+* Use [stackoverflow.com](http://stackoverflow.com) for specific questions about the software. Typically, these will be related to installation problems, errors and bugs. Development questions when forking the code are welcome as well. Use the `fiware-cosmos` tag.
+* Use [fiware-tech-help@lists.fi-ware.org](mailto:fiware-tech-help@lists.fi-ware.org) for general questions about the software. Typically, these will be related to the conceptual usage of the component, e.g. wether it suites for your project or not. It is worth to mention the issues reported to [fiware-tech-help@lists.fi-ware.org](mailto:fiware-tech-help@lists.fi-ware.org) are tracked under [http://jira.fiware.org](http://jira.fiware.org); use this Jira to see the status of the issue, who has been assigneed to, the exchanged emails, etc, nevertheless the answers will be sent to you via email too.
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/ba27b6cb37e70e5d3b30d25f63b9d5a19bcee92e